app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoParser.kt

```diff
@@ -37,7 +37,7 @@ object PocketVideoParser {
             title = jsonObj.getString("title"),
             // Note that the 'url' property of our domain object can be retrieved from
             // either of two JSON fields, and we make no distinction internally
-            url = jsonObj.optString("tv_url", null) ?: jsonObj.getString("url"),
```

Comment:
This is used by the server: we can't update it, right? My intention was that we'd update the `tv_url` objects in `pocket_videos.json` and the associated test file.